prisma/migrations/20240604213907_capture_method_changes/migration.sql

```diff
@@ -0,0 +1,66 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `capture_timestamp` on the `capture` table. All the data in the column will be lost.
+  - You are about to drop the column `release_timestamp` on the `capture` table. All the data in the column will be lost.
+  - Added the required column `capture_date` to the `capture` table without a default value. This is not possible if the table is not empty.
```

Comment:
Just checking: presumably not an issue for current critterbase dev/test/prod?

Reply:
Should be ok 👍🏼